tests/integration/gh-5792/pom.xml

```diff
@@ -52,12 +52,6 @@
         </dependency>
         <!-- Verify that the app builds and that config and OpenAPI document parsing work with the older SnakeYAML release. -->
         <dependency>
-            <exclusions>
```

Comment:
1. It turns out it was the explicit dependency on snakeyaml:1.32 in the test app's pom that was forcing the version to what we wanted; the exclusion I had included in that dep. was a no-op.
2. I have since pushed a commit which adds a dep. mgt section to the test app's pom to accomplish this instead of using a direct dep. We really do want any transitive dependency that uses SnakeYAML to use the older release (which the dep. mgt does nicely) and this project does not itself need a direct dep. on SnakeYAML.
3. Because both config-yaml and openapi depend on snakeyaml it would have been fragile to add the exclusion to this project's dep. on only one of them. Adding to both would have been "safer" perhaps but not totally safe; if some additional component added a dep. on SnakeYAML in the future that might have messed up this test project.